docs/source/jobs.rst

```diff
@@ -299,25 +401,12 @@ For example, to open the store in read-only mode, you would write:
     ...     print(job.data.x)
 
 Explicitly opening the underlying file by either using the context manager or the ``open()`` function is required when reading and writing arrays, such as ``numpy.arrays``.
-Please see the next section for details on accessing arrays.
+Please see the **Accessing arrays** section for details on accessing arrays.
```

Comment:
Can you use [`:ref:`](https://www.sphinx-doc.org/en/1.5/markup/inline.html#role-ref) to link to the referenced section?

Reply:
I'm having trouble linking the reference section. I tried to add the label above the "Accessing array headings" using `.. _accessing-arrays`:
```rst
.. _accessing-arrays:
Accessing arrays
----------------

All values stored in :attr:`job.data` are returned as copies, except for arrays, which are accessed *by reference* and not automatically copied into memory.
That is important to enable the storage of massive arrays that do not necessarily fit into memory.
```

Then add the reference like this it using `:ref:`
```rst
Please see the :ref:`accessing arrays <accessing-arrays>` section for details on accessing arrays.
```

But when I build the docs, the link does not take you to the correct section. The changes are in commit 49b837f

Reply:
May need a new line after the label.

Reply:
Reference links works now. Thanks to @vyasr for the tips!